lib/charms/mongodb/v0/mongodb.py

```diff
@@ -66,6 +69,14 @@ def uri(self):
         auth_source = ""
         if self.database != "admin":
             auth_source = "&authSource=admin"
+
+        if self.standalone:
```

Comment:
PBM was incorrectly configured previously (this only began causing issues in sharding) it should use the standalone URI